README.md

```diff
@@ -26,32 +26,29 @@ Features
 - Support for tomographic imaging with a unified geometry representation and bindings to external libraries for efficient computation of projections and back-projections.
 - Standardized tests to validate implementations against expected behavior of the corresponding mathematical object, e.g. if a user-defined norm satisfies `norm(x + y) <= norm(x) + norm(y)` for a number of input vectors `x` and `y`.
 
-Documentation
-=============
-
-Introduction, mathematical details and the API reference can be found at the [ODL documentation page](https://odlgroup.github.io/odl/).
-
-There is also a large number of examples available in the [examples](examples) folder.
-
 Installation
 ============
-For basic installation without extra dependencies, run
+Installing ODL should be as easy as
 
-    pip install odl
+    conda install odl
 
-You can check that everything was installed properly by running
+or
 
-    python -c "import odl; odl.test()"
+    pip install odl
 
-This requires [pytest](http://pytest.org/latest/). See the [installation](https://odlgroup.github.io/odl/getting_started/installing.html) documentation for further information.
+The code is compatible with Python 2 and 3 through the `future` library. It is intended to work on all major platforms (GNU/Linux / Mac / Windows).
 
-ODL is compatible to Python 2 and 3 through the `future` library. It is intended to work on all major platforms (GNU/Linux / Mac / Windows).
+Resources
+=========
+- [ODL Documentation](https://odlgroup.github.io/odl/)
+- [Installation guide](https://odlgroup.github.io/odl/getting_started/installing.html)
```

Comment:
Both is totally fine with me, i just dont want someone to try `pip install odl`, fail and then don't see the instructions on how to do more.